js/i18n.js

```diff
@@ -480,6 +480,99 @@ var msg = {
+		langName:   'Русский',
+		defCountry: 'us',
+		monthNames: [ 'Месяц', 'Январь', 'Февраль', 'Март', 'Апрель', 'Май', 'Июнь', 'Июль', 'Август', 'Сентябрь', 'Октябрь', 'Ноябрь', 'Декабрь' ],
+		weekDays:   [ 'Вс', 'Пн', 'Вт', 'Ср', 'Чт', 'Пт', 'Сб' ],
+		weekStart:  'Неделя начинается с',
+		sunday:     'Воскресенья',
+		monday:     'Понедельника',
+		year:       'Год',
+		month:      'Месяц',
+		day:        'День',
+		credits:    'Создано с <strong>desktopCal.js</strong>',
+		creditTitle:'Уведомление',
+		creditDescr:'You can customize this line to include your photo credits, for example.',
+		creditDescr:'Вы можете изменить эту строку, чтобы включить в нее информацию, например, об авторе фотографии.',
+		front:      'Сторона 1',
+		back:       'Сторона 2',
+		design:     'Выберите макет календаря',
+		edit:       'Выберите изображение, месяц и год',
+		language:   'Язык',
+		layout:     'Макет',
+		desktopCal: 'Настольный календарь',
+		wallSingle: 'Настенный календарь',
+		digitalBg:  'Цифровые обои',
+		screenConf: 'Конфигурация экрана',
+		screenRes:  'Разрешение экрана',
+		chgOrient:  'Изменить ориентацию',
+		calStyle:   'Стиль календаря',
+		small:      'Маленький блок',
+		medium:     'Средний блок',
+		large:      'Большой блок',
+		column:     'Вертикальная полоса',
+		row:        'Горизонтальная полоса',
+		calSettings: 'Настройки календаря',
+		modern:     'Современный',
+		classic:    'Классический',
+		showHolidays: 'Показывать описание праздников',
+		yes:        'Да',
+		no:         'Нет',
+		horAlign:   'Выравнивание по горизонтали',
+		verAlign:   'Выравнивание по вертикали',
+		left:       'Влево',
+		horCenter:  'По центру',
+		right:      'Вправо',
+		top:        'По верхнему краю',
+		verCenter:  'По центру',
+		bottom:     'По нижнему краю',
+		width:      'Ширина',
+		height:     'Высота',
+		pixels:     'пикселей',
+		rotateR:    'Повернуть по часовой стрелке',
+		rotateL:    'Повернуть против часовой стрелки',
+		flipH:      'Отразить по горизонтали',
+		flipV:      'Отразить по вертикали',
+		reset:      'Сбросить',
+		colors:     'Цвета',
+		colorPresets: 'Цветовые схемы',
+		saveColors: 'Сохранить как новую цветовую схему',
+		deletePreset: 'Вы действительно хотите удалить эту цветовую схему?\nЭТО ДЕЙСТВИЕ НЕЛЬЗЯ ОТМЕНИТЬ!',
+		bgColor:    'Цвет фона',
+		bgOpacity:  'Прозрачность фона',
+		textColor:  'Цвет текста',
+		holidayColor:'Цвет праздников',
+		loadImage:  'Загрузить изображение',
+		holidays:   'Праздники',
+		countryHolidays: 'Национальные праздники',
+		customHolidays: 'Свои праздники',
+		description: 'Описание',
+		none:       'Нет',
+		add:        'Добавить',
+		delete:     'Удалить',
+		load:       'Загрузить',
+		imgNotice:  'Изображения НЕ покидают ваш компьютер. Вся обработка происходит в вашем браузере.',
+		printIt:    'Напечатать!',
+		paperSize:  'Формат бумаги / соотношение сторон',
+		paperIso:   'A3 или A4',
+		paperLegal: 'Legal',
+		paperLetter:'Letter',
+		paperTabloid:'Tabloid',
```

Comment:
just checking.. no translation for these?

Reply:
No, they are apparently used just like that in printer settings, etc.